### PR TITLE
Add five backend options for simple model example

### PR DESCRIPTION
**Pull request by mingmingtasd**

### CODE REVIEW

**Comment by mingmingtasd** [comment]
@ibelem @Wenzhao-Xiang PTAL, thanks.

File: examples/simple/index.html [hunk 1]
```diff
@@ -3,7 +3,39 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>WebML Simple Model</title>
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.3/css/bootstrap.min.css">    
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.3/css/bootstrap.min.css">
```
Comment (ibelem) [inline examples/simple/index.html]: Please use https://github.com/intel/webml-polyfill/blob/master/examples/static/lib/bootstrap/css/bootstrap.min.css instead.

File: examples/simple/index.html [hunk 2]
```diff
@@ -3,7 +3,39 @@
...
+			border-radius:100%;
+			margin-top:3px;
+			/* margin-left:5px; */
+		
+		}
+		.radioHidden:checked + .radioInput{
+			border:2px solid #00a1e9;
+		}
+	  </style>
+    <!-- add -->
```
Comment (ibelem) [inline examples/simple/index.html]: What's the meaning of `<!-- add -->` here?

**Comment by mingmingtasd** [comment]
I have updated the code committed yesterday according to your suggestions. @ibelem

**Comment by mingmingtasd** [comment]
I have updated the code. @ibelem

**Comment by ibelem** [comment]
LGTM, thanks @mingmingtasd ! :)

**Pull request merged**